scripts/entrypoint.sh

```diff
@@ -9,6 +18,9 @@ echo "Wait for DB server to be ready"
 /opt/scripts/waitforit.sh "${DATABASE_HOST}:${DATABASE_PORT}"
 
 
-/usr/share/tomcat9/bin/catalina.sh start
+#/usr/share/tomcat9/bin/catalina.sh start
+/usr/share/tomcat9/bin/startup.sh
+
+func_fix_sql_migration&
```

Comment:
I do believe with out the '&' the script waits until the function is finished before executing the next command. In this case this would mean that the log would not be shown until the function finishes